packages/cactus-cmd-api-server/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:12.20.1-alpine3.12
```

Comment:
@jordigiam My PR adding this just got merged, should be safe to rebase onto main now and then this doesn't need to be in the diff here (and I'm guessing the other containerize PRs are going to be the same)

Reply:
Done